From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
Date: Mon, 29 Aug 2022 08:43:24 +0200
Subject: [ARCHEOLOGY] Add patches needed for Pi-KVM on Rpi4 (#4127)

> X-Git-Archeology: > recovered message: > * Add patches needed for Pi-KVM on Rpi4
> X-Git-Archeology: > recovered message: > * Remove patch that does not belong here
> X-Git-Archeology: > recovered message: > * Update config
> X-Git-Archeology: > recovered message: > * Update edge config
> X-Git-Archeology: - Revision 2fb3aaaf86a7463ef1d26f4d623e1d3289ea4f25: https://github.com/armbian/build/commit/2fb3aaaf86a7463ef1d26f4d623e1d3289ea4f25
> X-Git-Archeology:   Date: Mon, 29 Aug 2022 08:43:24 +0200
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Add patches needed for Pi-KVM on Rpi4 (#4127)
> X-Git-Archeology:
> X-Git-Archeology: - Revision acf5810cbe38c7578907ecd06e61abb0a446df1e: https://github.com/armbian/build/commit/acf5810cbe38c7578907ecd06e61abb0a446df1e
> X-Git-Archeology:   Date: Sat, 11 Mar 2023 19:19:13 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igor@armbian.com>
> X-Git-Archeology:   Subject: Move Rpi kernels to 6.2, move current to legacy (#4898)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 8caf51243e9769c48289be534f1ecc6aa4b53a15: https://github.com/armbian/build/commit/8caf51243e9769c48289be534f1ecc6aa4b53a15
> X-Git-Archeology:   Date: Sun, 19 Nov 2023 22:34:54 +0100
> X-Git-Archeology:   From: Igor <igor@armbian.com>
> X-Git-Archeology:   Subject: Bump rpi and ebin edge kernels to 6.6.y (#5925)
> X-Git-Archeology:
> X-Git-Archeology: - Revision f852beca0aeaec9ef5c062f089c3493e3b12fc48: https://github.com/armbian/build/commit/f852beca0aeaec9ef5c062f089c3493e3b12fc48
> X-Git-Archeology:   Date: Sun, 07 Jan 2024 18:58:36 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igor.pecovnik@gmail.com>
> X-Git-Archeology:   Subject: Switch meson64, bcm2711, sunxi, imx6, mvebu current to 6.6, edge to 6.7 and leave 6.1 for legecy
> X-Git-Archeology:
---
 drivers/usb/gadget/function/f_mass_storage.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/usb/gadget/function/f_mass_storage.c b/drivers/usb/gadget/function/f_mass_storage.c
index 111111111111..222222222222 100644
--- a/drivers/usb/gadget/function/f_mass_storage.c
+++ b/drivers/usb/gadget/function/f_mass_storage.c
@@ -2983,11 +2983,11 @@ void fsg_common_set_inquiry_string(struct fsg_common *common, const char *vn,
 	/* Prepare inquiryString */
 	i = get_default_bcdDevice();
 	snprintf(common->inquiry_string, sizeof(common->inquiry_string),
-		 "%-8s%-16s%04x", vn ?: "Linux",
+		 "%-8s%-16s%04x", vn ?: "PiKVM",
 		 /* Assume product name dependent on the first LUN */
 		 pn ?: ((*common->luns)->cdrom
-		     ? "File-CD Gadget"
-		     : "File-Stor Gadget"),
+		     ? "CD-ROM Drive"
+		     : "Flash Drive"),
 		 i);
 }
 EXPORT_SYMBOL_GPL(fsg_common_set_inquiry_string);
-- 
Armbian

